### DISCUSSION TITLE
Version bumping works locally, but not in gitlab-ci. Is there a good working example you can share?

### DISCUSSION THREAD

**Discussion by scollinstc** [callowayproject/bump-my-version#139] [Q&A]
Hello, I chose to use bump-my-version for a python package I developed. 
Everything works great with it locally--- however, I noticed if I use it in my gitlab-ci.yml pipeline, when I build using `pyproject.toml`, it's not picking up the version numbers and is setting them to 0.0.0. 

I am using simple semantic versioning and pretty much just copied and pasted the snippet from `bump-my-version sample-config --no-prompt --destination .bumpversion.toml` into pyproject.toml under the tag , after setting that my version is variable. 

The pertinent parts of the files look like this:
```
pyproject.toml

[project]
...
dynamic = ["dependencies", "version"]

[tool.bumpversion]
current_version = "0.3.0"
parse = "(?P<major>\\d+)\\.(?P<minor>\\d+)\\.(?P<patch>\\d+)"
serialize = ["{major}.{minor}.{patch}"]
search = "{current_version}"
replace = "{new_version}"
regex = false
ignore_missing_version = false
tag = false
sign_tags = false
tag_name = "v{new_version}"
tag_message = "Bump version: {current_version} → {new_version}"
allow_dirty = false
commit = false
message = "Bump version: {current_version} → {new_version}"
commit_args = ""
```

And for the pertinent part of the gitlab-ci file:
```
.gitlab-ci.yml

    - pip install twine bump-my-version
    - bump-my-version bump minor
    - pip install build
    - python -m build
    .... (uploads to gitlab package repository--- and works, except the pipeline fails right now because the version is coming in as 0.0.0)
```

What am I missing to make sure the new version number is actually available to the build process? Thanks!

NOTE: I'll be happy to do a pull request to the documentation with working examples addressing this once this question is answered, if desired.

**Comment by coordt** [comment]
The first thing I would suggest is changing `bump-my-version bump minor` to `bump-my-version bump -vv minor` in order to get more information.

Next, you didn't show any file configurations for places you want the version changed in your package. Do you have them?
